testbed/tests/scenarios.go

```diff
@@ -235,3 +237,48 @@ func Scenario1kSPSWithAttrs(t *testing.T, args []string, tests []TestCase, opts
 		})
 	}
 }
+
+// Structure used for TestTraceNoBackend10kSPS.
+// Defines RAM usage range for defined processor type.
+type ProcessorConfig struct {
```

Comment:
From what I see this is only used in this package. Let's make this unexported `processorConfig` since it does not need to be exported.